sound/soc/sof/Makefile

```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
 
 snd-sof-objs := core.o ops.o loader.o ipc.o pcm.o pm.o debug.o topology.o\
-		control.o trace.o iomem-utils.o sof-audio.o
+		control.o trace.o iomem-utils.o sof-utils.o sof-audio.o
```

Comment:
sof-test handles sof_utils:

git grep sof_utils
tools/kmod/sof_remove.sh:remove_module snd_sof_utils

Let's remove this patch.